README.md

```diff
@@ -20,9 +20,9 @@ The content dimension has three different views based on the grade range (elemen
 
 ## Installation for development
 
-The CR Tool will only work fully in the context of cfgov-refresh.
+The CR Tool will only work fully in the context of consumerfinance.gov.
 
-1. Clone this repository and install it locally into your cfgov-refresh environment, following [the cfgov-refresh docs for working on satellite apps](https://cfpb.github.io/cfgov-refresh/related-projects/#developing-python-packages-with-cfgov-refresh).
+1. Clone this repository and install it locally into your consumerfinance.gov environment, following [the consumerfinance.gov docs for working on satellite apps](https://cfpb.github.io/consumerfinance.gov/related-projects/#developing-python-packages-with-consumerfinance.gov).
```

Comment:
The anchor in this link does not include the `.`:

```suggestion
1. Clone this repository and install it locally into your consumerfinance.gov environment, following [the consumerfinance.gov docs for working on satellite apps](https://cfpb.github.io/consumerfinance.gov/related-projects/#developing-python-packages-with-consumerfinancegov).
```

Reply:
Fixed anchor link on line 25 of README.md